testsuite/MDAnalysisTests/test_distances.py

```diff
@@ -706,3 +706,9 @@ def test_missing_backend_raises_ValueError(self):
                                       backend="not_implemented_stuff")
         
         
+class TestOpenMP(object):
+    def test_serial(self):
+        assert_(MDAnalysis.lib._distances.OPENMP_ENABLED == False)
+
+    def test_serial(self):
```

Comment:
And yes, you could just test that it is either `True` or `False`.
Perhaps add a line to travis to explicitly echo what we are compiling by checking the flag.